users/common/strings.py

```diff
@@ -0,0 +1,44 @@
+from django.utils.translation import ugettext_lazy
+
+
+class CustomUserAdminText:
+    PERSONAL_INFO = ugettext_lazy('Personal info')
+    STATUS = ugettext_lazy('Status')
+    PERMISSIONS = ugettext_lazy('Permissions')
+    IMPORTANT_DATES = ugettext_lazy('Important dates')
```

Comment:
Somehow this is not good for me. Why `Important dates`?

Reply:
This is a commonly used name for this type of fields. At least that's what we had in the previous project. I thought that this is so irrelevant that I have omitted to change the name to another one.